### DISCUSSION TITLE
New patch release

### DISCUSSION THREAD

**Discussion by jdahm** [pydantic/pydantic#3436] [General]
#3403 fixed a support problem with the latest python releases. As a result codes using the latest release of `pydantic` cannot use these latest versions. Could you tag a new patch release incorporating this fix?

**Comment by silvanmelchior** [comment]
Are there any news about that?

**Comment by samuelcolvin** [comment]
I'm working on this, see https://twitter.com/samuel_colvin/status/1467456649468256258

**Reply by silvanmelchior** [reply]
Great to hear that, thanks a lot for all your efforts!

**Reply by jdahm** [reply]
Thanks for prepping this. It'll be nice not to have to pin the patch version of the interpreter in CI!
